mythril/analysis/analysis_args.py

```diff
@@ -0,0 +1,30 @@
+from mythril.support.support_utils import Singleton
+
+
+class AnalysisArgs(object, metaclass=Singleton):
+    """
+    This module helps in preventing args being sent through multiple of classes to reach analysis modules
+    """
+
+    def __init__(self):
+        self._loop_bound = 4
```

Comment:
The "ideal" default loop bound is 3 - best speed on SWC without false positives and false negatives